app/server.go

```diff
@@ -134,6 +135,14 @@ func (kr *KubeRouter) Run() error {
 	}
 
 	if kr.Config.RunRouter {
+
+		// enable GoBGP debug log
+		if kr.Config.EnableGoBGPLogging {
```

Comment:
Seems like it makes more sense for this to be a string flag that accepts the log level to use for gobgp (default `warn`) instead of just a bool.

Reply:
Good point. Will keep it as log level.